.github/workflows/event-merge-to-queue.yml

```diff
@@ -16,30 +16,24 @@ concurrency:
 # TODO: Use RedisJSON's `${{ vars.DEFAULT_REDISJSON_REF }}` branch when testing on nightly
 
 jobs:
-  get-latest-redis-tag:
-    uses: ./.github/workflows/task-get-latest-tag.yml
-    with:
-      repo: redis/redis
-      prefix: '8.0'
-
```

Comment:
don't delete it, why not just change line 23  unstable?

Reply:
There is no tag with the 'unstable' prefix

Reply:
When `8.0` is out we will want to remove the prefix input here, so reverting will be only the first step. Can we open a ticket with instructions + the PR number that needs to be reverted?

Reply:
Yes